webui/react/src/components/ResponsiveTable.module.scss

```diff
@@ -3,3 +3,6 @@
     color: var(--theme-colors-action-normal);
   }
 }
+.rightClickableCellWrapper {
+  display: inline-block;
```

Comment:
chore: I think this should be `display: inline`. inline-block makes it so even if you're not hovering over text, you sometimes can't open the context menu, while inline fits the amount of text present.

Reply:
Great thanks, will make that fix!